web/src/test/java/org/springframework/security/web/server/csrf/CsrfWebFilterTests.java

```diff
@@ -227,6 +229,21 @@ public void doFilterWhenTokenIsNullThenNoNullPointer() {
 				.isForbidden();
 	}
 
+	@Test
```

Comment:
I failed to mention earlier, will you please add `// gh-9113` to the top of this test for historical purposes?